packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -513,6 +513,9 @@ class ScrollableState<T extends Scrollable> extends State<T> with SingleTickerPr
   /// If there are no in-progress scrolling physics, this function scrolls to
   /// the given offset instead.
   void didUpdateScrollBehavior(double newScrollOffset) {
+    _contentExtent = scrollBehavior.contentExtent;
+    _containerExtent = scrollBehavior.containerExtent;
```

Comment:
I guess this makes sense.  Probably should have @Hixie review though given that his head is more into scrolling at the moment.